### DISCUSSION TITLE
Modal with scrollable view

### DISCUSSION THREAD

**Discussion by alejandrofdez-us** [ant-design/ant-design-mobile#3891] [Q&A]
I have a popup modal. This modal may show a FlatList or similar including a bunch of items. These items may not fit on the screen. How can I include a scroll view within the modal? I tried nesting the modal in ScrollView, or incoporating a ScrollView before the FlatList but it didn't work. Any solutions? Thanks

**Comment by awmleer** [comment]
Which version of `antd-mobile` are you using?

For v5 alpha, you can make the content of Popup scrollable like this:

<img width="438" alt="image" src="https://user-images.githubusercontent.com/16526078/129065501-327aa688-83bd-40f4-9bbb-ee161ff4902e.png">

Refer: https://next.mobile.ant.design/components/popup
(The documentation has not been translated yet. Sorry for the inconvenience.)
